webextension/scripts/settings.js

```diff
@@ -224,7 +223,8 @@ function addDocs () {
     'domaintools': 'Displays what Domaintools.com knows about the site you are on.',
     'wbmsummary': 'Displays what the Wayback Machine knows about the site you are on.',
     'annotations': 'Displays what Hypothes.is knows about the site you are on.',
-    'tagcloud': 'Show a Word Cloud built from Anchor text of links archived in the Wayback Machine.'
+    'tagcloud': 'Show a Word Cloud built from Anchor text of links archived in the Wayback Machine.',
+    'auto-update-context': 'Automatically update context windows when the page they are referencing changes.'
```

Comment:
Order change here was not required. !

Reply:
Yes, it wasn't required. I just wanted to keep items, belonging to the same menu, together :-)
Thank you!